app/workers/tipline_statistics_periodic_check_worker.rb

```diff
@@ -0,0 +1,13 @@
+class TiplineStatisticsPeriodicCheckWorker
+  include Sidekiq::Worker
+
+  sidekiq_options retry: 0
+
+  def perform
+    if CheckConfig.get('ENABLE_TIPLINE_STATS_MONITORING', true)
+      periodic_check = Time.now.ago(1.day)
```

Comment:
I would add some buffer here just to be safe (the batch job takes a while to start, etc.)

Reply:
Done make it 26 hours